ggml/src/ggml-cuda/common.cuh

```diff
@@ -319,6 +319,15 @@ static constexpr __device__ int ggml_cuda_get_physical_warp_size() {
 #endif // defined(GGML_USE_HIP) && (defined(__GFX9__) || defined(__GFX8__))
 }
 
+static constexpr __host__ int ggml_cuda_get_physical_warp_size_host() {
```

Comment:
You cant do this, you dont know what gpu it will be in host code at compile time. You have to gather the warp size from the ggml_cuda_device_info struct